docs/api/tray.md

```diff
@@ -85,11 +81,7 @@ Emitted when the tray icon is clicked.
   * `shiftKey` Boolean
   * `ctrlKey` Boolean
   * `metaKey` Boolean
-* `bounds` Object - the bounds of tray icon.
-  * `x` Integer
-  * `y` Integer
-  * `width` Integer
-  * `height` Integer
+* `bounds` Bounds - The [Bounds](structures/bounds.md) of tray icon
```

Comment:
Shouldn't the first mention of the type be the linkified one, instead of in the description? e.g. 

```
`bounds` [Bounds](structures/bounds.md) - The bound of tray icon
```